### ISSUE TITLE
tar.gz packages for Darwin have unexpected /Library dir

### ISSUE THREAD

**Issue by andrewkroh** [elastic/beats#8549]
The Elastic licensed tar.gz packages for darwin have an unexpected /Library directory.

```
$ tar tf ~/Downloads/metricbeat-6.5.0-SNAPSHOT-darwin-x86_64.tar.gz  | grep Library
/Library/Application Support/Elastic/metricbeat/bin/metricbeat
$ tar tf ~/Downloads/packetbeat-6.5.0-SNAPSHOT-darwin-x86_64.tar.gz | grep Libr
/Library/Application Support/Elastic/packetbeat/bin/packetbeat
$ tar tf ~/Downloads/metricbeat-oss-6.5.0-SNAPSHOT-darwin-x86_64.tar.gz | grep Lib
```

**Comment by andrewkroh** [comment]
I think I see the issue. This has the wrong path for a tar.gz package. Would you mind double-checking me on this @exekias.

https://github.com/elastic/beats/blob/5f1f94706641cda7cf9cf4c29fb8a1b181946d50/dev-tools/packaging/packages.yml#L360-L362

**Comment by exekias** [comment]
Ups, yes you are right, I think I mixed binary tar.gz with dmg distributions?

**Comment by andrewkroh** [comment]
This is now fixed in 6.5 and master.
